src/Pmi/Entities/Participant.php

```diff
@@ -69,6 +73,15 @@ public function parseRdrParticipant($participant)
             $this->status = false;
             $this->statusReason = 'withdrawal';
         }
+        if (!empty($this->genomicsStartTime) && isset($participant->consentForStudyEnrollmentAuthored) && $participant->consentForStudyEnrollmentAuthored > $this->genomicsStartTime) {
```

Comment:
Unlikely that there will be a consent time at the exact same time as the start time, but the ticket says this should be `>=`

```suggestion
        if (!empty($this->genomicsStartTime) && isset($participant->consentForStudyEnrollmentAuthored) && $participant->consentForStudyEnrollmentAuthored >= $this->genomicsStartTime) {
```